docs/edit/format.md

```diff
@@ -1,9 +1,10 @@
 System tests code is in python, and is linted/formated using [black](https://black.readthedocs.io/en/stable/) and [pylint](https://pylint.readthedocs.io/en/latest/).
 
-You'll only need [python3.12](https://www.python.org/downloads/) (you may have it by default) to run them, with a unique command :
+The only pre-req is: [python3.12](https://www.python.org/downloads/).
```

Comment:
We should ask users to install python3.12 via pyenv: https://github.com/pyenv/pyenv?tab=readme-ov-file#getting-pyenv. It's a great tool for managing multiple python versions and keeping system tests dependencies isolated to their own virtual environment.

Reply:
Can you share instructions for doing that?

Reply:
I would just link the pyenv docs and say this the recommended way of installing and managing multiple python versions on your machine (if they already have 3.12 then this isn't a concern). Using pyenv shouldn't be a requirement, just a suggestion. The only requirement is having 3.12.

Reply:
I think we want to call that out in the system-tests/README doc, not here. I'll add it there and then link to that doc inside this doc 👍